app/controllers/feedback/post-index.controller.js

```diff
@@ -19,18 +19,15 @@ ${req.body['feedback-suggestion']}`
     message: message
   }
 
-  zendeskClient.createTicket(opts)
-    .then(() => {
-      lodash.set(req, 'session.pageData.feedback', {})
-      req.flash('generic', 'Thanks for your feedback')
-      return res.redirect(paths.feedback)
-    }).catch(err => {
-      lodash.set(req, 'session.pageData.feedback', {
-        'feedback-suggestion': req.body['feedback-suggestion'],
-        'feedback-rating': req.body['feedback-rating']
-      })
-      logger.error(`Error posting request to Zendesk - ${err}`)
-      req.flash('genericError', 'We couldn’t send your feedback. Please try again')
-      return res.redirect(paths.feedback)
-    })
+  try {
+    await zendeskClient.createTicket(opts)
+
+    lodash.set(req, 'session.pageData.feedback', {})
+    req.flash('generic', 'Thanks for your feedback')
+    return res.redirect(paths.feedback)
+  } catch (err) {
+    logger.error(`Error posting request to Zendesk - ${err}`)
```

Comment:
This leaves unpacking the meaningful bit of the error to JavaScript which can mean getting some `[object Object]` fun in the logs. Recommend logging the errors message or the full error as context to the log
```suggestion
    logger.error(`Error posting request to Zendesk - ${err && err.message}`)
```
```suggestion
    logger.error('Error posting request to Zendesk', err)
```

Reply:
Thanks - I will make another PR for this change, I like the second option so will do that.